templates/root/prefilled/.acl

```diff
@@ -0,0 +1,13 @@
+@prefix acl: <http://www.w3.org/ns/auth/acl#>.
```

Comment:
We should add a comment on top, explaining what this file is and what it does, so people can always tell where it comes from.